contrib/change_group_leader.lua

```diff
@@ -0,0 +1,168 @@
+--[[
+  Passport cropping guide for darktable
+
+  copyright (c) 2021 Angel Angelov
+
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[
+  Original author: github.com/wpferguson
+]]
```

Comment:
You can delete this, the copyright comment above takes care of it, but thanks for the attribution.